### DISCUSSION TITLE
Windows please

### DISCUSSION THREAD

**Discussion by steelx** [cymian/mouseless#62] [General]
I really Want to purchase this, as I use Windows as well as MacOs.

- Few suggestions, need a hover mouse. and
- There is a glitch if mouse cursor is inside a Input field and even if Overlay is visible pressing Column grid characters actually types inside Input field rather than doing a click on Submit button

**Comment by cymian** [comment]
hi steelx, you'll be happy to know I'm doing technical evaluation for a Windows version :) (see #67)

can you explain more what you mean by hover mouse?

also, can you please fill out a bug report for your issue with the input field?  please include what app you're seeing it type into input fields in.  thank you!

**Comment by steelx** [comment]
thanks for windows evaluation. will post a bug in issue.

> hover

I mean there are certain button which gets activated after Mouse hover on it.

**Reply by cymian** [reply]
ah, you should be able to use the `hold_for_move` action, or `cycle_mouse_action` to move the mouse where you want to hover.    the [docs](https://github.com/croian/mouseless-issues/blob/main/mouseless_documentation.md) have more info if helps.  let me know if that works or if you have any more questions!

**Comment by cymian** [comment]
closed this, but please reopen / let me know if you have any more issues or questions, @steelx!
